<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubcategoryWindow</class>
 <widget class="QWidget" name="SubcategoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Subcategory</string>
   </property>
  </widget>
  <widget class="QTableView" name="subcategoryTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
